src/types.ts

```diff
@@ -1460,6 +1475,13 @@ export type Field = {
   value?: string;
 };
 
+export type FileUploadConfig = {
```

Comment:
why don't we use this type above ?

Reply:
was following the way that it's done for other objects, which repeat fields rather than using the type.

Reply:
any example for a reference ?

Reply:
Ah, you're right. I thought firebase_config and apn_config did this, but I see now they have slightly different fields in AppSettings. I'll fix that.

Reply:
Yes, they are different slightly but here is exactly same. Thanks for updating.